app/controllers/api/devices_controller.rb

```diff
@@ -140,6 +140,16 @@ def hive_queues
     render json: {}, status: status
   end
 
+  def screenshot
```

Comment:
Could you please add a check to see if `device.plugin` and `device.plugin.screenshot` exist and, if not, fail in a sensible way. Eg, a 405 (method not allowed) status.

For example, see https://github.com/bbc/hive_mind/blob/master/app/models/device.rb#L79